packages/api/src/jobs/send_email.ts

```diff
@@ -0,0 +1,37 @@
+import { env } from '../env'
+import { sendWithMailJet } from '../services/send_emails'
+import { Merge } from '../util'
+import { logger } from '../utils/logger'
+import { sendEmail } from '../utils/sendEmail'
+
+export const SEND_EMAIL_JOB = 'send-email'
+
+type ContentType = { html: string } | { text: string } | { templateId: string }
+export type SendEmailJobData = Merge<
+  {
+    emailAddress: string
+    subject?: string
+    html?: string
+    text?: string
+    templateId?: string
+    dynamicTemplateData?: Record<string, any>
+  },
+  ContentType
+>
+
+export const sendEmailJob = async (data: SendEmailJobData) => {
+  if (process.env.USE_MAILJET && data.dynamicTemplateData) {
+    return sendWithMailJet(data.emailAddress, data.dynamicTemplateData.link)
+  }
+
+  if (!data.html && !data.text && !data.templateId) {
```

Comment:
Do we require HTML and Text? Or should it just be one of them?

Reply:
The content should be one of them: `html` or `text` or `template`